### ISSUE TITLE
Run a fixed number of random tests

### ISSUE THREAD

**Issue by ykeselman** [jbasko/pytest-random-order#51]
Is there a way to limit the number of random tests to a fixed number? The scenario I'm looking at is a parameterized test that I want to execute with random sets of parameters a fixed number of times. Thanks!

**Comment by jbasko** [comment]
No, there isn't. This isn't about running random tests. This is only about randomizing the order and nothing else.
